src/Wpf/Prism.Wpf/Ioc/ContainerProviderExtension.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Windows.Markup;
+
+namespace Prism.Ioc
+{
+    /// <summary>
+    /// Provides Types and Services registered with the Container
+    /// </summary>
+    /// <example>
+    /// <para>
+    /// Usage as markup extension:
+    /// <![CDATA[
+    ///   <TextBlock
+    ///     Text="{Binding
+    ///       Path=Foo,
+    ///       Converter={prism:ContainerProvider {x:Type local:MyConverter}}}" />
+    /// ]]>
+    /// </para>
+    /// <para>
+    /// Usage as XML element:
+    /// <![CDATA[
+    ///   <Window>
+    ///     <Window.DataContext>
+    ///       <prism:ContainerProvider Type="{x:Type local:MyViewModel}" />
+    ///     </Window.DataContext>
+    ///   </Window>
+    /// ]]>
+    /// </para>
+    /// </example>
```

Comment:
if you see Microsoft Docs they show proper use of the example block inside of the summary block. https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments#code